.github/workflows/llm-translation-testing.yml

```diff
@@ -0,0 +1,122 @@
+name: LLM Translation Testing for Release Candidates
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_candidate_tag:
+        description: 'Release candidate tag/version'
+        required: true
+        type: string
+  push:
+    tags:
+      - 'v*-rc*'  # Triggers on release candidate tags like v1.0.0-rc1
+  
+jobs:
+  llm-translation-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 120
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.release_candidate_tag || github.ref }}
+      
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+      
+      - name: Cache pip dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install -r requirements.txt
+          pip install "pytest==7.3.1"
+          pip install "pytest-retry==1.6.3"
+          pip install "pytest-cov==5.0.0"
+          pip install "pytest-asyncio==0.21.1"
+          pip install "respx==0.22.0"
+          pip install "pytest-xdist==3.6.1"
+          pip install "pytest-html==4.1.1"
+      
+      - name: Create test results directory
+        run: mkdir -p test-results
+      
+      - name: Run LLM Translation Tests
+        run: |
+          python -m pytest -vv tests/llm_translation \
```

Comment:
instead of re-running the test can you fetch the results from circle ci. We already have this test running through there.